picasso/src/test/java/com/squareup/picasso/OkHttp3DownloaderTest.java

```diff
@@ -30,6 +25,12 @@
 import org.robolectric.RobolectricTestRunner;
 import org.robolectric.annotation.Config;
 
+import okhttp3.OkHttpClient;
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.MockWebServer;
+import okhttp3.mockwebserver.RecordedRequest;
+import okio.Okio;
```

Comment:
We have a 'SquareAndroid' style [here](https://github.com/square/java-code-styles) if you want to stop screwin' with our order 😝